mscore/preferences.cpp

```diff
@@ -693,6 +693,8 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
 
       connect(shortcutList,   SIGNAL(itemActivated(QTreeWidgetItem*, int)), SLOT(defineShortcutClicked()));
       connect(resetShortcut,  SIGNAL(clicked()), SLOT(resetShortcutClicked()));
+      connect(saveShortcutList,  SIGNAL(clicked()), SLOT(saveShortcutListClicked()));
+      connect(loadShortcutList,  SIGNAL(clicked()), SLOT(loadShortcutListClicked()));
```

Comment:
Still the UI for this is missing, so the build fails on Travis CI, see https://travis-ci.org/musescore/MuseScore/jobs/245227080#L2627-L2633. If it build for you, you must have added a saveSchortcutList and loadShortcutList button to prefsdialog.ui, but not added that to any commit yet.